app/src/main/java/com/github/onlynotesswent/model/note/Note.kt

```diff
@@ -60,101 +60,87 @@ data class Note(
       val publicPath: String
   )
 
-  /**
-   * Represents a list of Comments for a Note. The class is immutable. It's companion object is used
-   * to produce a new CommentCollection to get a new one.
-   */
+  /** Represents a list of Comments for a Note. The class is immutable. */
```

Comment:
For better readibility and consistency, put the doc in separate lines as follows: 

```
 /** 
  * Represents a list of Comments for a Note. The class is immutable.
  */
```